README.md

```diff
@@ -1,8 +1,15 @@
 # kind-image
 Kind-node image for the PowerPC architecture.
 
-Images are hosted at https://quay.io/repository/powercloud/kind-node?tab=tags and can be used as follows:
+Before spinning up the cluster, one may need to install the kind binary with the git patch applied to the repository 
+to use ppc64le specific images while setting up the cluster.
```

Comment:
I feel this is bit confusing, Is it possible to simplify, What does git patch applied to repository, What repository here?
If we want to manually patch, Can you link this to how to patch.